_posts/2020-01-27-why-upgrade-your-rails-application.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Why Is It Important to Upgrade Your Rails Application?"
+date: 2020-01-31 13:00:00
+categories: ["rails", "upgrades"]
+author: abizzinotto
+---
+
+Technology evolves quickly. When you first created your Rails application, it's likely you did it using the latest version of Rails. As time passes, however, new Rails versions come out and your application becomes more and more out of touch with the latest version. But why is this important? Why does it matter?
+
+There are several different reasons to upgrade your Rails application. In this article, I’ll list what we consider to be the most important ones.
+
+<!--more-->
+
+## Security
+
+Security is a big issue. You want your application to be as safe as possible and, to accomplish that, one of the pivotal things to do is to make sure Ruby on Rails is up to date. Ruby on Rails is the application framework underlying everything your web application does. 
```

Comment:
@ABizzinotto you can probably add this link https://www.fastruby.io/security-risks somewhere in this section

Reply:
Done